client-js/app/elements/app-link.html

```diff
@@ -1,5 +1,7 @@
 <script>
 (function () {
+  const MOUSE_LEFT_BUTTON = 0;
```

Comment:
Ahh, a good point. The mix of TypeScript and JavaScript trips me up constantly. I will change this to `var` and plan to transition JS to TS in separate pull requests as I encounter it.